account-kit/smart-contracts/src/ma-v2/utils.ts

```diff
@@ -0,0 +1,42 @@
+import { type Hex, concat } from "viem";
+
+export type PackSignatureParams = {
+  // orderedHookData: HookData[];
+  validationSignature: Hex;
+};
+
+// Signature packing utility
+export const packSignature = ({
+  // orderedHookData, TO DO: integrate in next iteration of MAv2 sdk
+  validationSignature,
+}: PackSignatureParams): Hex => {
+  return concat(["0xFF", "0x00", validationSignature]);
+};
```

Comment:
do all signature modules do this? or just `single-signer-validation`?

Reply:
this is a watered down implementation - when accounts have signature hooks that expect data, that data would be added here

Reply:
The account parses + validates the prefix in `validationSignature` (signature hooks that aren't here yet + the `FF` byte). So `packSignature` changes depending on the v0.8 ERC6900 account used

The module uses the `00` byte + `validationSignature` - this will change depending on the validation module used